docs/developers/developing.rst

```diff
@@ -14,7 +14,7 @@ Prerequisites
 You will need:
 
 * `git <https://git-scm.com/>`_
-* `Node.js <https://nodejs.org/en/>`_ v6.3+
+* `Node.js <https://nodejs.org/en/>`_ v12+
```

Comment:
Honestly, I'm not sure what version of node we now support, but I know 6.3 is super old. I think v12 should be safe?